blog_system/settings.py

```diff
@@ -0,0 +1,146 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+import private
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = private.SECRET_KEY

```

Comment:
I can't use the `python-dotenv` or equal because then github tests fail.
In `requirements.txt` i need to write `python-dotenv` but when i import `dotenv` module in code, github tests raise an error **"dotenv" module not found** because it is not in the `requirements.txt`